docs/external-template-creation.md

```diff
@@ -1,46 +1,34 @@
+- The (non-SVG) **image &amp; video files** used in each template is stored in [`public/static/main/images/templates/`](https://github.com/google/web-stories-wp/tree/static-site/public/static/main/images/templates) (in the `static-site` branch, using [Git LFS](https://git-lfs.github.com/)).
 
-To get the raw story json, simply open up a story in the web story editor with the devtools network tab open and press `save`. You should see the the story's json show up in the network tab. In that json, right click the `story_data` field and click `Store as global variable`. Go to the devtools console and type 
-
-```javascript
-copy(temp1)
-```
-
-When you press enter the story data should now be copied to your clipboard and ready to paste where you need.
-
-**Note**`Command+Shift+Option+J / Control+Shift+Alt+J` in the story editor opens up a dialog you can copy/paste raw story json to/from. However this is missing some fields `story_data` contains which are necessary for versioning the external templates.
-
-## Formatting Media Names In Story Creation
-
-WordPress has a naming convention where if you upload an image with the same name multiple times, it adds a `-x` suffix. 
-
-**i.e**
-if I upload `some_image.png` 3 times to the WordPress media upload in the story editor, it will store those images as:
-
-```javascript
-// some_image.png
-// some_image-1.png
-// some_image-2.png
-```
+## Overview
 
-Because of this, our conversion script will strip the `-x` suffix off of the image source so it can point to the correct image stored in our repo.
+To add a new template to the editor:
 
-This is an important note to make when naming images for upload into a story as **a `-x` suffix will always be stripped**. This naming convention is reserved for WordPress and will break your template creation if you use it in your image naming.
+1. [Engineer] Commit all SVGs used in the template to the codebase as stickers ([details](#adding-svgs-to-the-codebase-as-stickers)).
+2. [Designer] Create a new story in your local WP environment and replicate the template design.
+    - SVGs should already be available in the Stickers panel from step (1) &mdash; click on a sticker to insert it into the story. If you can't find the Stickers Panel, make sure the `enableStickers` experiment is turned on.
+    - Upload (drag & drop) images and videos into the story to add them.
+3. [Engineer] Commit all images & videos used in the template to the codebase.
+    - Make sure images are not too large &mdash; full-width images should be 1080p, large images should be 720p, and small images should be 480p. See [#6485](https://github.com/google/web-stories-wp/pull/6485) as an example.
+    - Make sure videos are 720p.
+4. [Engineer] Get the story JSON ([details](#get-the-story-json)) and commit it to the codebase.
+    - Replace image & video URLs in the JSON with the `replaceme.com` placeholder.
+5. [Both] Verify that new template shows up in the template library and looks as expected.
```

Comment:
We should add a note about naming convention here.

In current templates we use naming convention like `travel_page9_image1.jpg`, `wellbeing_page10_image4.png`, `wellbeing_page4_bg.jpg` etc.